src/Libs/OwlCore.WinUI/Behaviors/OpenUriAction.cs

```diff
@@ -9,16 +9,16 @@ namespace OwlCore.WinUI.Behaviors
     /// <summary>
     /// A behavior action that opens the provided parameter as a Uri.
     /// </summary>
-    public class OpenUriAction : DependencyObject, IAction
+    public partial class OpenUriAction : DependencyObject, IAction
     {
-        public Uri? Source { get; set; }
+        public string? Source { get; set; }
```

Comment:
If its just a string, maybe rename the property to something like `SourceAddress` or `SourceString` or `UriPath`